snapcraft/setup-etc

```diff
@@ -0,0 +1,15 @@
+#!/bin/sh
+set -eu
+
+# Re-exec outside of apparmor confinement
```

Comment:
We shouldnt need to do this in this script as the callers should have already done it.

Reply:
Fixed.